### ISSUE TITLE
swagger-parser options interface

### ISSUE THREAD

**Issue by sandro-pasquali** [PayU/api-schema-builder#52]
I'd like to be able to set options for `swagger-parser` (https://apitools.dev/swagger-parser/docs/options.html). Is that possible? I don't see where to do that.

Great tool. Thanks.

**Comment by kibertoad** [comment]
@sandro-pasquali Sounds useful. I'll give it a shot.

**Comment by sandro-pasquali** [comment]
@kibertoad Yes, I think so too. Although I realize that since schema-builder accepts a JSON object the implementation can parse swagger any way it likes and pass that along. Still, if you don't want to, or can't, do that...
